src/main/java/Duke/Task.java

```diff
@@ -0,0 +1,35 @@
+package Duke;
+
+public class Task {
+    protected String description;
+    protected boolean isDone;
+    private static char NOT_DONE_SYMBOL = '\u2718';
+    private static char DONE_SYMBOL = '\u2713';
+
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+
+    public String getDescription() {
```

Comment:
Good job. Clean and clear naming of methods

Reply:
Thank you for your compliment! :)